lib/ratgdo/log.h

```diff
@@ -21,7 +21,7 @@ void print_packet(uint8_t pkt[SECPLUS2_CODE_LEN]);
 // This can be large, but not too large.  IRAM heap is approx 18KB, we also need
 // space for other data in here, so during development monitor logs and adjust
 // this smaller if necessary.  IRAM malloc's are all done during startup.
-#define LOG_BUFFER_SIZE 8192
+#define LOG_BUFFER_SIZE 1024
```

Comment:
Let's put this back to 8192 and test.  If it fails, lets try 4096, etc til we find one that works.

Reply:
I tested with 8192, it seems to be working fine. When would you expect a fail if it were to happen?

Reply:
It would fail immediately during startup. So I think safe to put back.